WorkflowUI/Sources/Container/ContainerViewController.swift

```diff
@@ -27,22 +27,27 @@
             return workflowHost.output
         }
 
-        internal let rootViewController: DescribedViewController
+        private(set) var rootViewController: UIViewController
```

Comment:
Sort of: The contained VC type is now the type of the workflow's rendered screen, not a `DescribedViewController`, since we're removing a layer of nesting. However this var was always internal, so the only way it'll affect folks is if they're reaching upwards from the contained VC, or downwards into the Container, in a non-resilient way.

Reply:
i apologize, i should have been more specific. i was wondering about the access control changes & `let` vs `var` in particular. this didn't look like it was being set anywhere but init, so wondering if this could just be `private let`, or maybe just `let` to keep test visibility.

Reply:
It is set in the update method, via `update(child: \.rootViewController, with: screen, in: environment)` (the key path) if the underlying VC type has to change.

Reply:
aah, right it was a writable keypath. thanks for clarifying!